migtests/tests/pg/assessment-report-test/pg_assessment_report.sql

```diff
@@ -294,3 +294,56 @@ END;
 $$ LANGUAGE plpgsql;
 
 select process_order(1);
+
+-- In PG migration from pg_dump the function parameters and return will never have the %TYPE syntax, instead they have the actual type in the DDLs
+-- e.g. for the below function this will be the export one `CREATE FUNCTION public.process_combined_tbl(p_id integer, p_c cidr, p_bitt bit, p_inds3 interval) RETURNS macaddr`
+CREATE OR REPLACE FUNCTION public.process_combined_tbl(
```

Comment:
In end-to-end tests, let's only add a very basic case (with one instance of `%TYPE` in a function and procedure. This will help us avoid changing these expected files frequently (which can be a pain). 

Let's write unit tests (we can test `GetAllIssues` function) for all other cases (func params, func return, etc).

Reply:
same for the other analyze-schema test.

Reply:
For assessment, its mainly a basic case only getting tested, as pg_dump doesn't give `%TYPE` syntax in parameters and return types, so those are not getting tested but I thought its better to have it here in case pg_dump behavior changes anytime.

For analyze yes, we can do that I can take that up later when we will add unit tests for `GetAllIssues` to remove them from here.

Reply:
Even for assessment, I think end-to-end tests should only test the most basic thing: Are `% TYPE` issues getting reported or not? 
The other cases can be unit tested with `GetAllIssues` . 
That way, less things to change in the expectedFiles as well! 
Okay if you want to take it up later 👍